AK/Assertions.cpp

```diff
@@ -136,7 +156,6 @@ static AssertionHandlerFunc get_custom_assertion_handler()
 #    pragma clang diagnostic ignored "-Wcast-function-type-mismatch"
         auto handler = reinterpret_cast<AssertionHandlerFunc>(GetProcAddress(module, "ak_assertion_handler"));
 #    pragma clang diagnostic pop
-        FreeLibrary(module);
```

Comment:
You remove this `FreeLibrary()` call in this commit, but don't mention it in the commit message. This should probably be a separate commit with the message you put in your PR description.